### PR TITLE
Migrate chat.expensify.com to expensify.cash

### PR DESCRIPTION
**Pull request by Jag96**
@AndrewGable @coleaeason will you please review this
cc @rafecolton 

This PR updates usages of `chat.expensify.com` to `expensify.cash`. This PR is on hold to prevent chat downtime, once the rest of the steps from https://github.com/Expensify/Expensify/issues/147628 are done/ready we can merge this.

### Fixed Issues
This resolves steps 1 and 2 for https://github.com/Expensify/Expensify/issues/147628#issuecomment-744999420.

### Tests
1. I will test this once the following items are done (cc @coleaeason in case testing this relies on other steps)
```
- Set up expensify.cash as a CNAME to cloudfront like we currently have for chat.expensify.com
- Add expensify.cash as an Alternate Domain Name to the cloudfront distribution, and request a new certificate with ACM
```

### CODE REVIEW

**Review by coleaeason — DISMISSED** [review]

**Review by rafecolton — DISMISSED** [review]
LGTM. Going to hold on merging this until we're ready to pull the trigger.

**Review by rafecolton — APPROVED** [review]
LGTM after fixing merge conflicts, will merge as soon as tests pass

**Pull request merged**
